lib/peopleHR.ts

```diff
@@ -71,6 +72,81 @@ async function parseXml(xmlString: string): Promise<any> {
     );
 }
 
+/**
+ * Pulls the PeopleHR RSS feed data from the cache,
+ * or fetches it from the feed if the cache has expired
+ * @returns JSON of the RSS feed
+ */
+async function getJobPostingData() {
+    const cacheFilePath = path.join(
+        process.cwd(),
+        'files',
+        'peopleHRcache.json',
+    );
+
+    if (existsSync(cacheFilePath)) {
+        const cacheFile = readFileSync(cacheFilePath, 'utf8');
+        const cacheJSON = JSON.parse(cacheFile);
+        const cacheTTL = 10 * 1000; //One hour 60 * 60 * 1000
+
+        if (cacheJSON.lastUpdated + cacheTTL < Date.now()) {
+            // The cache has timed out, fetch new data from PeopleHR
+
+            const peopleHRJobPostingsXML = await fetchPeopleHRFeed();
```

Comment:
**Suggestion**: It looks like we're not catching any server errors, might be best to wrap this in a try catch statement as well instead of using `if (peopleHRJobPostingsXML === null)`